### ISSUE TITLE
Calling the parent click handler

### ISSUE THREAD

**Issue by m-ibm** [alexandersimoes/d3plus#534]
Is it possible to invoke the parent's click behavior when providing a `click` function?
E.g., when the special `d3plus_other` is clicked I would like to default behavior.

```javascript
.mouse({
	"click" : function(dp, d3viz) {
		if (dp.my_id == "d3plus_other") {
			return true;
		} else if 
```
Currently we've worked around this by returning true/false and changing D3plus to run its click code on `true`. I would very much like to avoid changing anything in the original library though.

**Comment by davelandry** [comment]
If I'm understanding this correctly: Do you want to be able to invoke the default click behavior, given a certain condition?

**Comment by m-ibm** [comment]
Yes, under some condition I do not want to provide behavior, default is fine (e.g. zooming into the **other** rectangle in a tree-map)

**Comment by davelandry** [comment]
closed in favor of #447
